witness/src/Dscp/Snowdrop/Expanders.hs

```diff
@@ -33,10 +35,11 @@ expandBlock ::
        , HasLens ctx RestrictCtx
        , HasLens ctx (ChgAccumCtx ctx)
        )
-    => Block
+    => FeeCoefficients
```

Comment:
Didn't I ask to use reflection? Or there's a reason not to do that?

Reply:
We have Genesis block application, which has to be not covered by fees, so reflection is used in `Logic.Processing`.